cmd/kubeadm/app/util/apiclient/wait.go

```diff
@@ -244,6 +244,7 @@ func getStaticPodSingleHash(client clientset.Interface, nodeName string, compone
 	staticPodName := fmt.Sprintf("%s-%s", component, nodeName)
 	staticPod, err := client.CoreV1().Pods(metav1.NamespaceSystem).Get(context.TODO(), staticPodName, metav1.GetOptions{})
 	if err != nil {
+		fmt.Printf("[apiclient] Get Static Pod %s failed with err: %v\n", staticPodName, err)
```

Comment:
It will print a lot of failed logs if we add the log here. 
For instance, WaitForStaticPodControlPlaneHashes will check it every 500ms and the wait will wait 5 mins. That means 600 lines of this log.

Before the waiting failed, we will log something like `Waiting for the kubelet to restart the component`. I would suggest to add more info there if it is not enough.

Reply:
Thanks for reminder！But actually time out err will return in early func `WaitForStaticPodControlPlaneHashes` and won't print the message you mentioned.

I tried make it print err message only if  err is different from lastErr, it can reduce much more log.